etc/state-migration-test/Cargo.lock

```diff
@@ -708,7 +708,7 @@ dependencies = [
 [[package]]
 name = "rjson"
 version = "0.3.2"
-source = "git+https://github.com/aurora-is-near/rjson#cc3da9495e7e520900d66d2b517539f74fff93cf"
+source = "git+https://github.com/aurora-is-near/rjson?rev=cc3da949#cc3da9495e7e520900d66d2b517539f74fff93cf"
```

Comment:
Shouldn't it be something like

```suggestion
source = "git+https://github.com/aurora-is-near/rjson?rev=cc3da9495e7e520900d66d2b517539f74fff93cf"
```

Reply:
The lock file is automatically generated so I can only assume it's right as is

Reply:
Ok, it didn't generate this change for me, but I'm ok with it :)